samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/package.json

```diff
@@ -12,10 +12,10 @@
     "@testing-library/user-event": "^13.5.0",
     "fluid-framework": "~0.59.0",
     "jquery": "^3.6.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "17.0.0",
+    "react-dom": "17.0.0",
```

Comment:
Why did you downgraded the packages ?
Ideal way is to use latest versions.

Reply:
Resolved dependency with other package and restored latest version